cypress/integration/step_definitions/backend/at_usersubscripton.cy.js

```diff
@@ -0,0 +1,27 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+let res;
+Given('User should get refresh_token and create authToken', function () {
+  cy.generateToken ()
+  });
+
+When('User sends a get request to usersubscription endpoint', () =>{
+  cy.userSubscription().then((response)=>{
+    expect(response.status).to.eq(200)
+    res = response;
+  })
+})
+And('User should be able to see info of all subscription types', () =>{
+  expect(res.body.email).to.eq("abdullatif.aksu@abeyis.com")
```

Comment:
Can you read this data from feature file? We don't want hard coded test data in the test files. "abdullatif.aksu@abeyis.com" email and other values should be read from feature file or fixture file.